### PR TITLE
fix jupyter components, add jupyter back to TOC

### PR DESCRIPTION
**Pull request by MykolaGolubyev**
image

image

### CODE REVIEW

**Comment by MykolaGolubyev** [comment]
@jyeakel this should be a stop on my side. Please review.

**Comment by MykolaGolubyev** [comment]
There is probably more work that needs to be done, but let's throw this out.

File: znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js [hunk 1]
```diff
@@ -17,11 +17,22 @@
...
         <div className="jupyter-cell jupyter-text content-block">
-            <elementsLibrary.CliOutput lines={text.split('\n')}/>
+            <elementsLibrary.CliOutput lines={lines}/>
         </div>
     )
 }
 
+function convertToLines(text) {
+    const lines = text.split('\n')
+    if (lines.length > 0 && lines[lines.length - 1].length === 0) {
```
Comment (byc1234) [inline znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js]: Don't you need to lines.pop() for all of empty lines? For example, don't you need to make `xyx\n\n\n\n\n` to `xyz'?

Comment (MykolaGolubyev) [reply znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js]: You are right, I was handling only natural last line line ending, but it makes sense to trim it fully. Updated.

Comment (byc1234) [reply znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js]: Do you still need `convertToLines()` to handle a natural last line ending?

Comment (MykolaGolubyev) [reply znai-reactjs/src/doc-elements/jupyter/JupyterTextCell.js]: duh... changed :)

**Pull request merged**